packages/plugin-assets-retry/src/runtime/index.ts

```diff
@@ -278,7 +286,18 @@ function resourceMonitor(
 
 // @ts-expect-error init is a global function, ignore ts(6133)
 function init(options: PluginAssetsRetryOptions) {
-  const config = Object.assign({}, defaultConfig, options);
+  const config: PluginAssetsRetryOptions = {};
+
+  for (const key in defaultConfig) {
```

Comment:
why changed to `for in` here

Reply:
We didn't add polyfill for `Object.assign` and our runtime code does not support es5